app/forms/concerns/steps/applicant_or_partner.rb

```diff
@@ -0,0 +1,17 @@
+module Steps
+  module ApplicantOrPartner
+    extend ActiveSupport::Concern
+
+    def subject
+      I18n.t('dictionary.subject', ownership_type:)
+    end
+
+    def ownership_type
+      if include_partner_in_means_assessment?
```

Comment:
I'm getting errors locally on the land/commercial/residential forms saying that this method cannot be found. I think that's fixed by including the type of means assessment module in this module